appium/tests/screenobjects/inbox.screen.ts

```diff
@@ -29,10 +29,8 @@ class InboxScreen extends BaseScreen {
   }
 
   clickOnEmailBySubject = async (subject: string) => {
+    await browser.pause(2000); // todo: loading inbox. Fix this: wait until loader gone
     const selector = `~${subject}`;
-    if (await (await $(selector)).isDisplayed() !== true) {
-      await TouchHelper.scrollDown();
-    }
```

Comment:
This will still be needed, else other tests will start failing. Some messages are on the bottom of the list, and the app needs to scroll to see them. (until fixed with #1071 )

The adding of a pause was to make sure that initial messages can be loaded, based on which `if (await (await $(selector)).isDisplayed() !== true) {` can make a proper decision whether to scroll or not.

Reply:
In some cases I don't need to scroll. So I can keep this one and create additional method without scrolling, OK?

Reply:
In the cases when you don't need to scroll, `if (await (await $(selector)).isDisplayed() !== true) {` should recognize that scroll is not needed. And therefore it should not scroll. Can you confirm?

I know this is a mess as it is, I'm just trying to not make it even worse.